autoload/multiple_cursors.vim

```diff
@@ -317,6 +325,17 @@ function! s:Cursor.remove_visual_selection() dict
   let self.visual_hi_id = 0
 endfunction
 
+" Restore unnamed register from paste buffer
+function! s:Cursor.save_paste_buffer() dict
```

Comment:
I think I would rename these to match the comments (so this one becomes `restore_paste_register` and the other becomes `save_paste_register`) but you might disagree